website/docs/quickstart/deploy/deployment-quick.md

```diff
@@ -10,7 +10,7 @@ sidebar_label: 单机部署
 
 ## 服务部署
 
-- 下载 [taier源码包](https://github.com/DTStack/Taier/releases/download/${current.version}/taier.tar.gz)
+- 下载 [taier源码包](https://github.com/DTStack/Taier/releases/download/v1.3.0/taier.tar.gz)
```

Comment:
Wouldn't it be better to link directly to the release page, rather than the tag page? Let users choose their own release version to deploy.